rust/examples/gen_assigned_numbers.rs

```diff
@@ -0,0 +1,98 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! This "example" doesn't show how to use the bumble API, but instead generates Rust code with
```

Comment:
would it make sense to put this under a `tools` subdir (like `bumble/tools/generate_company_id_list.py`) rather than `examples`?

Reply:
Done.